Gemfile.lock

```diff
@@ -50,9 +51,10 @@ PLATFORMS
 
 DEPENDENCIES
   pry-byebug (~> 3.5)
+  rake
   rspec (~> 3.6)
   simplecov-rcov (~> 0.2)
   synchronised_migration!
 
 BUNDLED WITH
-   1.16.1
+   1.17.2
```

Comment:
Surely this should be 2.0.x

Reply:
Previous issue but easy fix, This file should be .gitignored